### PR TITLE
Wfprev-71 Create wildfire header for Prevention app

### PR DESCRIPTION
**Pull request by yzlucas**
https://apps.nrs.gov.bc.ca/int/jira/browse/WFPREV-71
Adding generic header for prevention application. The logout and IDIR display will be implemented after https://apps.nrs.gov.bc.ca/int/jira/browse/WFPREV-82 is completed.

Unit tests and storybook added
image

### CODE REVIEW

**Comment by yzlucas** [comment]
1 test failed, going to fix it before merge.

File: client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-header/app-header.component.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+
+  environment:string = 'DEV'
+  title:string = 'PREVENTION'
+  currentUser: string = 'User_1'
+
+  onLogoutClick(){
+    //clear token will be implemented after authorization piece done
+    // this.tokenService.clearLocalStorageToken();
```
Comment (agordon-vivid) [inline client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-header/app-header.component.ts:35]: Are we removing this intentionally?  Can we just delete vs. comment out?

**Review by agordon-vivid — APPROVED** [review]

File: client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-footer/app-footer.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>app-footer works!</p>
```
[the remark this answers is not in the document]
Comment (yzlucas) [reply client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-footer/app-footer.component.html:1]: Ok, I thought it was a pending task. I justhave this footer created but not being used in the app

File: client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-header/app-header.component.ts (hunk 1 above)
Comment (yzlucas) [reply client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-header/app-header.component.ts:35]: We will need this function once we have the authentication piece done to log out the user, so I think we better leave this in place.

**Pull request merged**